pkgs/by-name/ra/radicle-desktop/package.nix

```
@@ -0,0 +1,124 @@
+{
+  pkgs,
+  cargo-tauri,
+  fetchgit,
+  git,
+  glib,
+  gtk3,
+  importNpmLock,
+  lib,
+  libsoup_3,
+  nodejs,
+  openssh,
+  openssl,
+  pkg-config,
+  playwright-driver,
+  radicle-node,
+  rustPlatform,
+  webkitgtk_4_1,
+  wrapGAppsHook4,
+}:
+
+let
+  twemoji-assets = pkgs.fetchFromGitHub {
+    owner = "twitter";
+    repo = "twemoji";
+    rev = "v14.0.2";
+    hash = "sha256-YoOnZ5uVukzi/6bLi22Y8U5TpplPzB7ji42l+/ys5xI=";
+  };
+in
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "radicle-desktop";
+  version = "unstable-2025-08-15";
```

Why not follow package versioning?, do you need the unstable (2 newer commits) for something?

They keep track of main version on `crates/radicle-tauri/tauri.conf.json`
[v0.8.0](https://app.radicle.xyz/nodes/seed.radicle.xyz/rad:z4D5UCArafTzTQpDZNQRuqswh3ury/commits/aeb405aaf53b56a426ab8d68c7f89b8953683224)

Also, see official [PKGBUILD](https://app.radicle.xyz/nodes/seed.radicle.xyz/rad:z4D5UCArafTzTQpDZNQRuqswh3ury/tree/arch/radicle-desktop/PKGBUILD) to confirm it.

```suggestion
  version = "0.8.0";
```


Note: for unstable versioning prefix the latest version if it exist, else a 0
[Versioning](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)
```suggestion
  version = "0.8.0-unstable-2025-08-15";
```

---

Ah, I didn't see that they use that instead. The `Cargo.toml` files are empty that's why I chose unstable.